src/arrays.js

```diff
@@ -1,59 +1,164 @@
 export const getNthElement = (index, array) => {
-  // your code here
+  
+if(index >= array.length) {
+  
+  	return array[index%(array.length)]
+}
+  else {
+
+    return array[index]
+  }
+}
 };
 
 export const arrayToCSVString = (array) => {
-  // your code here
+  return array.join()
 };
 
 export const csvStringToArray = (string) => {
-  // your code here
+  return string.split(",")
 };
 
 export const addToArray = (array, element) => {
-  // your code here
+  
+return array.push(element)
+	
 };
 
 export const addToArray2 = (array, element) => {
-  // your code here
+  array.push(element)
+	return array
 };
 
 export const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index,1)
 };
 
 export const numbersToStrings = (numbers) => {
-  // your code here
+  return numbers.toString().split(",")
 };
 
 export const uppercaseWordsInArray = (strings) => {
-  // your code here
+let stringsUpper = []
+	for(let i = 0; i < strings.length; i++) {
+    	stringsUpper.push(strings[i].toUpperCase())
+    	}
+  return stringsUpper
 };
 
 export const reverseWordsInArray = (strings) => {
-  // your code here
+let arr2 = []
+for (let i = 0; i < strings.length; i++)  
+  
+ {
+  
+ 	let str = ""
+ 
+ 	for (let j = strings[i].length - 1; j >=0;  j--)  
+  
+   		{
+    
+     			str += (strings[i][j]) ;      
+     
+   		}    
+   
+   	arr2.push(str) ;  
+  
+ }
+
+return arr2
+
 };
 
 export const onlyEven = (numbers) => {
-  // your code here
+const even = num => {
+	return num % 2 === 0;
+};
+
+const myEvenArray = numbers.filter(even);
+	return(myEvenArray)
 };
 
 export const removeNthElement2 = (index, array) => {
-  // your code here
+	const newArr = [...array]
+	newArr.splice(index,1)
+  		return newArr
 };
 
 export const elementsStartingWithAVowel = (strings) => {
-  // your code here
+  let vowelsArray =[]
+
+
+for (let i =0; i < strings.length; i++) {
+        if(strings[i][0] === "a" || strings[i][0] === "e" || strings[i][0] === "i" 
+         || strings[i][0] === "o" || strings[i][0] === "u" || strings[i][0] === "A" || strings[i][0] === "E" || strings[i][0] === "I" 
+         || strings[i][0] === "O" || strings[i][0] === "U") {
+     
+     vowelsArray.push(strings[i])
+     
+     }  
+    
+}
+  
+return (vowelsArray)
+
 };
 
 export const removeSpaces = (string) => {
-  // your code here
+  string = string.replace(/\s/g,'');  
+
+return string
 };
 
 export const sumNumbers = (numbers) => {
-  // your code here
+
+  let sum = numbers.reduce((a, b) => a + b, 0)
+
+    return sum
 };
 
 export const sortByLastLetter = (strings) => {
-  // your code here
+let arr2 = []
+	for (let i = 0; i < strings.length; i++)  
+  
+ {
+  
+ 	let str = ""
+ 
+ 		for (let j = strings[i].length - 1; j >=0;  j--)  
+  
+   		{
+    
+     			str += (strings[i][j]) ;      
+     
+   		}    
+   
+   	arr2.push(str) ;  
+  
+ }
+
+let arr3 = (arr2.sort())
+
+let arr4 = []
```

Comment:
Try and go for meaningful variable names like `sortedArray` rather than `arr3` and `arr4`. This makes your code more readable for other developers.